### ISSUE TITLE
Update for v2.23.0

### ISSUE THREAD

**Issue by mustaqsheikh** [Bixie/intellij-uikit#3]
Thanks for your contribution, do you have an update for v2.23.0 planned ?

**Comment by malles** [comment]
No, not in fact. There are no new components or snippets to add in this version.
If I'm overlooking something, please let me know.
